src/style.css

```diff
@@ -1,3 +1,3 @@
 body {
-  background-color: purple;
-}
\ No newline at end of file
+  font-family: Cambria, serif;
+}
```

Comment:
- [ ] You need to style your page, so that it looks the same as the wireframe provided in the projects description
### Your version
![Screenshot from 2022-10-11 23-40-16](https://user-images.githubusercontent.com/56886064/195203822-f06aa79c-a1f3-4bfd-a24c-7e9d9808fc43.png)

### Wireframe in project description
![Screenshot from 2022-10-11 23-43-07](https://user-images.githubusercontent.com/56886064/195203961-083dbaab-0059-497d-ba59-33b591777a18.png)

Reply:
Done! (only layout according to the requirements for this milestone) 😄 

https://github.com/Camilovelag/Leaderboard/blob/7d09f602259723ef9445ab6a89463c1d72b9746a/src/style.css#L1-L7

![image](https://user-images.githubusercontent.com/106117541/195207615-4d9133e0-c77d-4cac-b9fd-9f551227d28d.png)